styles.css

```diff
@@ -89,16 +89,53 @@ article {
     width: 80%;
 }
 
+main {
+    overflow: auto;
+}
+
 a {
     text-decoration: none;
     color: #F77913;
 }
 
+.recipeContainer {
+    width: 50%;
+    padding: 10px;
+    float: left;
+}
+
+.recipe {
+    width: 100%;
+    overflow: auto;
+    background-color: #FFFFFF;
+    border: 1px solid #F77913;
+    border-radius: 5px;
+    padding: 10px;
+    float: left;
+}
+
+.recipe div {
+    float: left;
+    padding: 0 5px;
+}
+
+.recipe img {
+    float: left;
+}
+
+.recipe a {
+    display: block;
```

Comment:
Suggest adding margin: 15px;

Reply:
styled this button slightly differently so that it doesn't stretch across the screen to make it more like the buttons on index page. Used 15px margin on top and bottom.